src/app/server/app.go

```diff
@@ -60,51 +68,68 @@ func NewApp(opts ...AppOptions) *App {
 		),
 	}
 
-	// Define routes and middlewares
-	app.Engine.StaticFS("/static", &assetfs.AssetFS{
-		Asset:    Asset,
-		AssetDir: AssetDir,
-		Prefix:   "static",
-	})
+	// Define static dir
+	app.Engine.Static("/static", "/client")
```

Comment:
I thought I nailed that one because I've tested the app and the static files seemed to be served, I will keep working on this one in order to get go-bindata back on track.